yt_dlp/extractor/glomex.py

```diff
@@ -205,8 +205,6 @@ def _extract_urls(cls, webpage, origin_url):
             mdict = mobj.groupdict()
             if mdict.get('url'):
                 url = unescapeHTML(mdict['url'])
-                if url.startswith('//'):
-                    url = f'https:{url}'
```

Comment:
This is needed since the valid_url has `https?` in this case

Reply:
In `_extract_urls` scheme is optional, as we use `EMBED_RE`, which incorporates `VALID_SRC`, which builds upon `_BASE_PLAYER_URL_RE`.